src/logging/logniWrapper.js

```diff
@@ -1,24 +1,34 @@
 // @flow
 import logni from 'logni.js'
 
+/**
+ * Prepends widget info in front of a message.
+ *
+ * @param msg
+ * @returns {string}
+ */
+const addWidgetInfoTo = (msg: string, widgetType: string) => {
+  return `(${widgetType}) ${msg}`
+}
+
 /**
  * Simple wrapper of logni.
  */
 const logniWrapper = {
-  info: (msg: string, level: number) => {
-    window.logni.info(msg, level)
+  info: (msg: string, level: number, widgetType: string) => {
```

Comment:
Toto je potreba vymyslet trochu jinak. Pointa wrapperu je v tom, abys nemusel porad predavat nejakou pevne danou informaci (pr. cart id v kosiku, nebo tady ten widget type, pripadne jine ID, ktere muzou mit jine widgety). Protoze to uz bys ji rovnou mohl pridat do `msg`.

V kosiku se to nacitalo z cookie. Tady bude potreba tu informaci predat v kontruktoru widgetu a potom ji nejak sikovne predavat do logni ;) Moznosti bude urcite vic. Promysli to, kdyztak si zavolame.